dev-env/bootstrap/teams.yaml

```diff
@@ -0,0 +1,94 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: greenhouse.sap/v1alpha1
+kind: Team
+metadata:
+  name: test-team-1
+  namespace: test-org
+spec:
+  description: Test Team 1
+  mappedIdPGroup: TEST_TEAM_1
+---
+apiVersion: greenhouse.sap/v1alpha1
+kind: TeamMembership
+metadata:
+  name: test-team-1
+  namespace: test-org
+spec:
+  members:
+  - id:        "01234"
+    firstName: "Jane"
+    lastName:  "Doe"
+    email:     "jane.doe@sap.com"
+  - id:        "12345"
+    firstName: "John"
+    lastName:  "Doe"
+    email:     "john.doe@sap.com"
+  - id:        "23456"
+    firstName: "Max"    
+    lastName:  "Mustermann"
+    email:     "max.mustermann@sap.com"
+---
+apiVersion: greenhouse.sap/v1alpha1
+kind: Team
+metadata:
+  name: test-team-2
+  namespace: test-org
+spec:
+  description: Test Team 2
+  mappedIdPGroup: TEST_TEAM_2
+---
+apiVersion: greenhouse.sap/v1alpha1
+kind: TeamMembership
+metadata:
+  name: test-team-2
+  namespace: test-org
+spec:
+  members:
+  - id:        "01234"
+    firstName: "Jane"
+    lastName:  "Doe"
+    email:     "jane.doe@sap.com"
```

Comment:
Could you rename the email addresses to `@example.com`?

Reply:
:100: